variables.tf

```diff
@@ -379,6 +418,27 @@ variable "os_disk_type" {
   nullable    = false
 }
 
+variable "outbound_ip_address_ids" {
+  type        = list(any)
+  description = "(Optional) The ID of the Public IP Addresses which should be used for outbound communication for the cluster load balancer."
+  default     = null
+  nullable    = true
+}
+
+variable "outbound_ip_prefix_ids" {
+  type        = list(any)
```

Comment:
According to the [provider code](https://github.com/hashicorp/terraform-provider-azurerm/blob/307e6478cf507c41301413475d77574618031db5/internal/services/containers/kubernetes_cluster_resource.go#L817-L827) here, the type should be `set(string)` here.